Sources/TracingInstrumentation/InstrumentationSystem+Tracing.swift

```diff
@@ -0,0 +1,42 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift Tracing open source project
+//
+// Copyright (c) 2020 Moritz Lang and the Swift Tracing project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import Baggage
+import Instrumentation
+
+
+extension InstrumentationSystem {
+
+    /// Get an `TracingInstrument` instance of the given type.
```

Comment:
On the iPad, so no suggestion: "an" should be "a".

Reply:
thx

Reply:
```suggestion
    /// Get a `TracingInstrument` instance of the given type.
```